stubs/prettytable/prettytable/prettytable.pyi

```diff
@@ -141,6 +145,62 @@ class PrettyTable:
     def oldsortslice(self): ...
     @oldsortslice.setter
     def oldsortslice(self, val) -> None: ...
+    @property
+    def bottom_junction_char(self) -> Incomplete: ...
```

Comment:
I think we can just leave this unannotated rather than doing an explicit `-> Incomplete` here. Same goes for most of the other properties in this class that you've added.

```suggestion
    def bottom_junction_char(self): ...
```